packages/wallet/test/SingleAddressWallet/load.test.ts

```diff
@@ -130,10 +138,18 @@ const assertWalletProperties2 = async (wallet: ObservableWallet) => {
 describe('SingleAddressWallet load', () => {
   it('loads all properties from provider, stores them and restores on subsequent load, fetches new data', async () => {
     const stores = createInMemoryWalletStores();
-    const wallet1 = await createWallet(stores, mocks.mockWalletProvider(), mocks.mockUtxoProvider());
+    const wallet1 = await createWallet(stores, {
+      rewardsProvider: mocks.mockRewardsProvider(),
+      utxoProvider: mocks.mockUtxoProvider(),
+      walletProvider: mocks.mockWalletProvider()
+    });
     await assertWalletProperties(wallet1, somePartialStakePools[0].id);
     wallet1.shutdown();
-    const wallet2 = await createWallet(stores, mocks.mockWalletProvider2(100), mocks.mockUtxoProvider2(100));
+    const wallet2 = await createWallet(stores, {
+      rewardsProvider: mocks.mockRewardsProvider2(100),
+      utxoProvider: mocks.mockUtxoProvider2(100),
+      walletProvider: mocks.mockWalletProvider2(100)
+    });
     await assertWalletProperties(wallet2, somePartialStakePools[0].id);
     await waitForWalletStateSettle(wallet2);
```

Comment:
This method is what makes the tests timeout. But not sure why

Reply:
Was it fixed in ad3209143fee1180ed4f3c0793331f1c6ba5628c?

Reply:
That's correct